README.md

```diff
@@ -19,10 +19,28 @@
    This will start a webpack-dev-server instance at port 8080 and start a
    watcher that will rebuild the browser extension with every file change.
 
-1. In Firefox 57 + open `about:debugging` and load the
-   `build/extension/manifest.json` file.
+1. To load the extension locally you can do this in the following ways:
 
-1. Visit `http://localhost:8080` to see the web-based theme editor - changes
+	  - In Firefox 57 + open `about:debugging` and load the
+	      `build/extension/manifest.json` file.
+
+	  - run the following steps:
+
+	  	```
+	    	rm -rf addon.xpi
+	    	npm run package
```

Comment:
`rm -rf addon.xpi` is not needed, `npm run package` automatically takes care of that.

`npm run start` is not required either. `npm run package` creates the XPI and outputs it to `addon.xpi`.

Reply:
I thought I had to restart the server but just teseted this, thanks! 👍